test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -469,6 +486,37 @@ UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_tag_offload_selection, self_rcx,
                               "self,rc_x");
 
 
+class test_ucp_tag_offload_cuda : public test_ucp_tag_offload {
+public:
+    test_ucp_tag_offload_cuda() {
+        modify_config("RNDV_THRESH", "1024");
+    }
+};
+
+// Test that expected SW RNDV request is handled properly when receive buffer
+// is allocated on GPU memory.
+UCS_TEST_P(test_ucp_tag_offload_cuda, sw_rndv_to_cuda_mem, "TM_SW_RNDV=y")
+{
+    activate_offload(sender());
+
+    size_t size   = 2048;
+    ucp_tag_t tag = 0xCAFEBABE;
```

Comment:
minor: `0xCAFEBABElu`? Coverity can complain on it - let's wait for the results

Reply:
it should be ok